rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -201,10 +206,19 @@ def get_airdrop_data(name: str, data_dir: Path) -> Tuple[Iterator, TextIO]:
             request = requests.get(url=AIRDROPS[name][0], timeout=DEFAULT_TIMEOUT_TUPLE)
         except requests.exceptions.RequestException as e:
             raise RemoteError(f'Airdrops Gist request failed due to {str(e)}') from e
-
-        with open(filename, 'w') as f:
-            f.write(request.content.decode('utf-8'))
-
+        try:
+            content = request.content.decode('utf-8')
```

Comment:
you can simply do `request.text`. Also replace `request` with `response`. The name seems to be wrong here.